packages/hash/api/src/model/knowledge/linkEntity.model.ts

```diff
@@ -179,19 +179,20 @@ export default class extends EntityModel {
     graphApi: GraphApi,
     params: {
       properties: PropertyObject;
-      linkOrder: EntityLinkOrder;
+      leftOrder?: number;
+      rightOrder?: number;
       actorId: string;
     },
   ): Promise<EntityModel> {
-    const { properties, actorId, linkOrder } = params;
+    const { properties, actorId, leftOrder, rightOrder } = params;
 
     const { data: metadata } = await graphApi.updateEntity({
```

Comment:
This update can throw an error if the graph layer `"cannot set left order of a link that does not have a left order"`. Do we want to catch for that axios error here? Currently it results in the axios response object being passed to the user:
```json
{
  "error": {
    "errors": [
      {
        "message": "Converting circular structure to JSON\n    --> starting at object with constructor 'ClientRequest'\n    |     property 'res' -> object with constructor 'IncomingMessage'\n    --- property 'req' closes the circle",
        "extensions": {
          "code": "INTERNAL_SERVER_ERROR",
          "exception": {
            "stacktrace": [
              "TypeError: Converting circular structure to JSON",
              "    --> starting at object with constructor 'ClientRequest'",
              "    |     property 'res' -> object with constructor 'IncomingMessage'",
              "    --- property 'req' closes the circle",
              "    at JSON.stringify (<anonymous>)",
              "    at prettyJSONStringify (/home/ahmad/dev/hash/hash/node_modules/apollo-server-core/src/runHttpQuery.ts:573:15)",
              "    at processHTTPRequest (/home/ahmad/dev/hash/hash/node_modules/apollo-server-core/src/runHttpQuery.ts:458:14)",
              "    at runMicrotasks (<anonymous>)",
              "    at processTicksAndRejections (node:internal/process/task_queues:96:5)"
            ]
          }
        }
      }
    ]
  }
}
```

Reply:
The happens when, for example, adding an order to a `Membership` link entity

Reply:
We discussed this offline, it seems that we want to have a general way to catch these errors, such as what we've outlined in https://app.asana.com/0/1201095311341924/1202945471553579/f